src/client/src/orangehrmLeavePlugin/pages/LeaveRequest.vue

```diff
@@ -111,6 +111,7 @@ import usei18n from '@/core/util/composable/usei18n';
 import useDateFormat from '@/core/util/composable/useDateFormat';
 import {formatDate, parseDate} from '@/core/util/helper/datefns';
 import useLocale from '@/core/util/composable/useLocale';
+import useEmployeeNameTranslate from '@/core/util/composable/useEmployeeNameTranslate';
```

Comment:
please dev test this screen and confirm working

Reply:
Working